src/app/functions/server/getExperiments.ts

```diff
@@ -29,11 +41,14 @@ export async function getExperiments(
         method: "POST",
         body: JSON.stringify({
           client_id: userId,
-          context: { key: "example-key" },
+          context: {
+            locale: session?.user.fxa?.locale,
```

Comment:
Any chance `session.user.` could be undefined/null and needs `user?.`?
Or I guess if it was, it'd trigger the `try..catch` wrapper and log an error about being unable to connect to Cirrus and log the exception to Sentry.